extensions/interactions/TextInput/directives/text-input-validation.service.spec.ts

```diff
@@ -22,43 +22,56 @@ import { AnswerGroup, AnswerGroupObjectFactory } from
   'domain/exploration/AnswerGroupObjectFactory';
 import { AppConstants } from 'app.constants';
 import { InteractionSpecsConstants } from 'pages/interaction-specs.constants';
-import { OutcomeObjectFactory } from
-  'domain/exploration/OutcomeObjectFactory';
-import { Rule, RuleObjectFactory } from
-  'domain/exploration/RuleObjectFactory';
-import { SubtitledUnicode } from
-  'domain/exploration/SubtitledUnicodeObjectFactory';
-import { TextInputValidationService } from
-  'interactions/TextInput/directives/text-input-validation.service';
+import { Outcome, OutcomeObjectFactory } from 'domain/exploration/OutcomeObjectFactory';
+import { Rule, RuleObjectFactory } from 'domain/exploration/RuleObjectFactory';
+import { SubtitledUnicode } from 'domain/exploration/SubtitledUnicodeObjectFactory';
+import { TextInputValidationService } from 'interactions/TextInput/directives/text-input-validation.service';
+import { TextInputCustomizationArgs } from 'interactions/customization-args-defs';
 
 describe('TextInputValidationService', () => {
-  let validatorService, WARNING_TYPES;
-  let INTERACTION_SPECS, customizationArgSpecs, rowsSpecs, minRows, maxRows;
-
-  let currentState, customizationArguments;
-  let goodAnswerGroups, goodDefaultOutcome;
-  let oof, agof, rof;
+  let validatorService: TextInputValidationService;
+  let WARNING_TYPES = AppConstants.WARNING_TYPES;
+  let INTERACTION_SPECS = InteractionSpecsConstants.INTERACTION_SPECS;
+  let minRows: number | undefined;
+  let maxRows: number | undefined;
```

Comment:
Why are these undefined?

Reply:
Added comment, basically for testing purposes